src/Promitor.Integrations.AzureMonitor/AzureMonitorClient.cs

```diff
@@ -7,6 +7,7 @@
 using Microsoft.Azure.Management.Monitor.Fluent.Models;
 using Microsoft.Azure.Management.ResourceManager.Fluent;
 using Microsoft.Azure.Management.ResourceManager.Fluent.Authentication;
+using Newtonsoft.Json;
```

Comment:
I don't think we need this?

Reply:
I'm using it to serialize a time series object into JSON, for better printing/logging. Please advice there's a better way though, I'm new to C#/Promitor.

Reply:
It's OK - Only saw this use later on